src/go/rdctl/pkg/utils/utils.go

```diff
@@ -0,0 +1,77 @@
+package utils
+
+import (
+	"io/fs"
+	"os"
+	"path"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/directories"
+)
+
+// Get the parent (or grandparent, or great-grandparent...) directory of fullPath.
+// numberTimes is the number of steps to ascend in the directory hierarchy.
+func MoveToParent(fullPath string, numberTimes int) string {
+	fullPath = path.Clean(fullPath)
+	for ; numberTimes > 0; numberTimes-- {
+		fullPath = path.Dir(fullPath)
+	}
+	return fullPath
+}
+
+/**
+ * Verify the path exists. For Linux pass in mode bits to guarantee the file is executable (for at least one
+ * category of user). Note that on macOS the candidate is a directory, so never pass in mode bits.
+ * And mode bits don't make sense on Windows.
+ */
+func CheckExistence(candidatePath string, modeBits fs.FileMode) string {
```

Comment:
I have comments about the name (or return type) of the function and how `modeBits` is used, but given that you just moved it, it seems better left in a new PR :)

Reply:
I agree, there are a couple improvements that could be made to this code. Another PR 😁